src/cephadm/cephadm

```diff
@@ -45,7 +45,7 @@ from pathlib import Path
 FuncT = TypeVar('FuncT', bound=Callable)
 
 # Default container images -----------------------------------------------------
-DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:master'
+DEFAULT_IMAGE = 'quay.ceph.io/ceph-ci/ceph:quincy'
```

Comment:
It seems that in pacific we're using `v16` tag: https://github.com/ceph/ceph/blob/pacific/src/cephadm/cephadm#L51-L51